File: servidor/modelos/consultorio.js
var mongoose = require('mongoose');
var Schema = mongoose.Schema;
const AutoIncrement = require('mongoose-sequence')(mongoose);

var consultorioSchema = new Schema({

	nomeConsultorio:{type: String,required:true},
	rua:{type: String,required:true},
	referencia:{type: String,required:false},
  	bairro:{type: String,required:true},
  	cidade:{type: String,required:true},
  	uf:{type: String,required:true},
  	cep:{type:Number,	required:true},
  	numero:{type:Number,	required:true}

});

consultorioSchema.plugin(AutoIncrement, {inc_field: 'idConsultorio'});

var consultorio = mongoose.model('Consultorio',consultorioSchema);

module.exports = consultorio;

File: MedicoAqui/app/src/main/java/com/example/gabriela/medicoaqui/Activity/Entities/Consultorio.java
package com.example.gabriela.medicoaqui.Activity.Entities;

public class Consultorio {

    String nomeConsultorio, rua, referencia, bairro, cidade, uf, cep, numero;

    public Consultorio(String nomeConsultorio, String rua, String referencia, String bairro, String cidade, String uf, String cep, String numero){
        nomeConsultorio = this.nomeConsultorio;
        rua = this.rua;
        referencia = this.referencia;
        bairro = this.bairro;
        cidade = this.cidade;
        uf = this.uf;
        cep = this.cep;
        numero = this.numero;

    }

    public String getNomeConsultorio() {
        return nomeConsultorio;
    }

    public void setNomeConsultorio(String nomeConsultorio) {
        this.nomeConsultorio = nomeConsultorio;
    }

    public String getRua() {
        return rua;
    }

    public void setRua(String rua) {
        this.rua = rua;
    }

    public String getReferencia() {
        return referencia;
    }

    public void setReferencia(String referencia) {
        this.referencia = referencia;
    }

    public String getBairro() {
        return bairro;
    }

    public void setBairro(String bairro) {
        this.bairro = bairro;
    }

    public String getCidade() {
        return cidade;
    }

    public void setCidade(String cidade) {
        this.cidade = cidade;
    }

    public String getUf() {
        return uf;
    }

    public void setUf(String uf) {
        this.uf = uf;
    }

    public String getCep() {
        return cep;
    }

    public void setCep(String cep) {
        this.cep = cep;
    }

    public String getNumero() {
        return numero;
    }

    public void setNumero(String numero) {
        this.numero = numero;
    }

}
